src/layouts/components/Editable/AddSectionButton.tsx

```diff
@@ -2,7 +2,15 @@ import { HStack, Icon, MenuItemProps, Text } from "@chakra-ui/react"
 import { Menu, MenuButtonProps } from "@opengovsg/design-system-react"
 import { BiPlus } from "react-icons/bi"
 
-export const AddSectionButton = ({ children, ...rest }: MenuButtonProps) => {
+interface AddSectionButtonProps extends MenuButtonProps {
+  sectionType?: string
+}
+
+export const AddSectionButton = ({
+  children,
+  sectionType = "section",
```

Comment:
think zj made a change here to put the prop as `buttonText` - more inclined to just use `buttonText` cos less restrictive

Reply:
Oh yeah that works better, changed in https://github.com/isomerpages/isomercms-frontend/pull/1457/commits/6c91fb77d62f110ead8d875b0bce976159cb64cc